src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -225,6 +225,10 @@ void HexagonGame::initLua_Utils()
         .doc(
             "Force-swaps (180 degrees) the player when invoked. If `$0` is "
             "`true`, the swap sound will be played.");
+
+    addLuaFn("u_getVersion", //
+        [this] { return Config::getVersionString(); })
+        .doc("Returns the string representing the current version of the game");
```

Comment:
Instead of this, how about the functions:

- `u_getVersionMajor`
- `u_getVersionMinor`
- `u_getVersionMicro`

which would return integers, and then from the Lua side instead of doing string manipulation we could just do some basic int comparisons? The Lua function could accept `(major, minor, micro)` as arguments

Reply:
We could keep this one as `u_getVersionString` in case anyone wants that